### DISCUSSION TITLE
umi4中antd日期组件是英文的怎么切换成中文呀

### DISCUSSION THREAD

**Discussion by Wxwya** [umijs/umi#11771] [Q&A] [answered]
求大佬回答

**Comment by fz6m** [comment, marked as answer]
从 antd 文档寻找一下吧，比如 [DatePicker 国际化配置](https://ant.design/components/date-picker-cn#%E5%9B%BD%E9%99%85%E5%8C%96%E9%85%8D%E7%BD%AE)

**Reply by Wxwya** [reply]
找了呀 没用 设置出来年是正常的 月份和星期还是英文的

**Reply by trampboy** [reply]
@Wxwya 可能还缺了 `dayjs.locale('en');`
你具体可以看下 [ConfigProvider全局化配置代码](https://ant-design.antgroup.com/components/config-provider-cn#config-provider-demo-theme)

如果你用的不是 day.js 而是 moment.js，则需要配置 moment.js。
也可以参考下我最近迁移到 Umi 4.0 整理的一些信息，[将一个老项目的 Web 应用框架升级为 Umi 4.0](https://juejin.cn/post/7291935944118026266)
